scripts/install

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
What are the scripts for? Can we add notes in readme for usage?

Reply:
this is for publishing to rubygems - though i dont think we really need them for this gem. ill just remove